.github/labeler.yml

```diff
@@ -0,0 +1,25 @@
+# https://github.com/actions/labeler#common-examples
+# Adapted from https://github.com/rapidsai/cudf/blob/main/.github/CODEOWNERS
+# Labels culled from https://github.com/rapidsai/cugraph/labels
+
+python:
+  - python/**
+  - notebooks/**
+  
+cuGraph:
+  - cpp/**
+
+CMake:
+  - **/CMakeLists.txt
+  - **/cmake/**
+  
+Java:
+  - java/**
+  
+Ops:
+  - .github/**
+  - /ci/**
```

Comment:
I'm assuming this is boilerplate we don't need to review, but the leading `/` here for the `ci` folder looked suspicious.